### ISSUE TITLE
Train-dev-test splits

### ISSUE THREAD

**Issue by Mahhos** [google-research/bert#1031]
Hi. Is there any way to use 3 datasets of train, dev, test for training, calculating hyperparameters, and evaluating the model? It seems that the code only uses `train.tsv` form training, and `dev.tsv` for evaluating the model. I need to use a separate validation set for computing hyperparameters and loss function. At the end, I will do prediction on an unlabeled dataset.  So, I have 4 datasets (train-dev-test labeled, and one unlabeled dataset).
